bundles/framework/personaldata/PublishedMapsTab.js

```diff
@@ -275,7 +275,55 @@ Oskari.clazz.define('Oskari.mapframework.bundle.personaldata.PublishedMapsTab',
             }
             return gridModel;
         },
-
+        _isCurrentProjectionSupportedForView: function (data) {
+            var currentProjection = Oskari.getSandbox().getMap().getSrsName();
+            var viewProjection = data.state.mapfull.state.srs;
+            if ( viewProjection !== currentProjection ) {
+                return false;
+            }
+            return false;
+        },
+        constructUrlUuid: function (srs) {
+            var views = Oskari.app.getSystemDefaultViews();
+            var uuid;
+            views.forEach( function (view) {
+                if ( view.srsName === srs ) {
+                    uuid = view.uuid;
+                    return;
+                }
+            });
+            var url = window.location.origin;
+            if (window.location.pathname && window.location.pathname.length) {
+                url += window.location.pathname;
+            }
+            url += "?uuid="+uuid;
+            return url;
+        },
+        _loadProjectionUuid: function (url) {
+            window.location.href = url;
```

Comment:
This is not used anywhere?

Reply:
Oh, it is. Maybe you should just wrap this to a function when used or at least move the implementation closer to where it's used.

Reply:
See the comment below. I would remove this function.